Jellyfin.Server/Filters/SecurityRequirementsOperationFilter.cs

```diff
@@ -6,86 +6,72 @@
 using Microsoft.OpenApi.Models;
 using Swashbuckle.AspNetCore.SwaggerGen;
 
-namespace Jellyfin.Server.Filters
+namespace Jellyfin.Server.Filters;
+
+/// <summary>
+/// Security requirement operation filter.
+/// </summary>
+public class SecurityRequirementsOperationFilter : IOperationFilter
 {
-    /// <summary>
-    /// Security requirement operation filter.
-    /// </summary>
-    public class SecurityRequirementsOperationFilter : IOperationFilter
+    private const string DefaultAuthPolicy = "Default";
```

Comment:
This was previously called `DefaultAuthorization`, I can change it easily in the SDK but perhaps we want to stay consistent?